lua/claudecode/config.lua

```diff
@@ -29,7 +29,11 @@ M.defaults = {
     { name = "Claude Opus 4 (Latest)", value = "opus" },
     { name = "Claude Sonnet 4 (Latest)", value = "sonnet" },
   },
-  terminal = nil, -- Will be lazy-loaded to avoid circular dependency
+  terminal = {
+    provider_opts = {
+      external_terminal_cmd = nil, -- Command template for external terminal (e.g., "alacritty -e %s")
+    },
+  },
```

Comment:
This should remain as it was; the `M.defaults` will be loaded from the terminal.lua file when `apply` is called.


```suggestion
  terminal = nil, -- Will be lazy-loaded to avoid circular dependency
```